core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -241,8 +240,8 @@ protected void tryComplete() {
 
   protected synchronized void tryTerminate() {
     if (this.state == STATE_COMPLETE && (externalCompletion == null || externalCompletion.isDone())) {
-      if (LOGGER.isDebugEnabled()) {
-        LOGGER.debug(this + " terminated.");
+      if (debugLogEnabled) {
+        LOGGER.debug("{} terminated.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: No need to call "toString()" method as formatting and string conversion is done by the Formatter. ([squid:S3457](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWXotIXnmsUmglC&open=AWkLjRWXotIXnmsUmglC)

[ISSUE_KEY]: AWkLjRWXotIXnmsUmglC